pages/results/gov-wide-data.html

```diff
@@ -42,3 +42,15 @@ <h3>Subject matter experts</h3>
   {{ smes[0].count }} data scientists from eleven agencies came together to evaluate applicants to the Data Scientist positions. 
 </p>
 
+<h3>Lesson Learned: types of data people</h3>
+<ul>
+ <li>Data Analysts help you understand and describe your data.</li>
+ <li>Data Scientists help build models to explain your data or predict things about data and outcomes you haven't seen before, and can help you scope data.</li>
+   <ul>
+      <li>Some data scientists specialize in Causal Inference -- they help you understand root causes of things. They are especially good for tasks like A/B testing, social science-y questions, and power calculations.</li>
+      <li>Some data scientists specialize in Machine Learning -- they help you make predictions about data you haven't seen before. They are especially good at building predictive models. </li>
+      <li>Some data scientists are on the cutting edge of Deep Learning or Generative Modeling -- they can help with "easy for humans, hard for computers" problems, like writing an essay or drawing a picture. </li>
+   </ul>
+ <li>Data Engineers help you build tooling to process and manage large volumes of data.</li>
```

Comment:
I might change this a little bit? _Data Engineers_ help build data pipelines and data products, often for use in data science and analysis